src/python_testing/TC_EEVSE_Utils.py

```diff
@@ -97,6 +157,9 @@ async def send_test_event_trigger_charge_demand(self):
     async def send_test_event_trigger_charge_demand_clear(self):
         await self.send_test_event_triggers(eventTrigger=0x0099000000000005)
 
+    async def send_test_event_trigger_time_of_use_mode(self):
```

Comment:
We should add a time_of_use_mode_clear as well

Reply:
Added but will be fully implemented when issue #34249 is addressed.